_pages/professional-services.md

```diff
@@ -0,0 +1,80 @@
+Most of the time we are doing sessions of 1 to 3 days, with raising level of technical experience each day.
+The training is always held by two of our MISP core team.
+
+The MISP team gives frequently open trainings within their premises in Luxembourg and sometimes abroad.
+A list of current trainings can be found at the [events page](/events/).
+
+There are different ways of custom trainings:
+
+a) For on-site closed trainings up to 30-50 attendees (depending on conditions of the location): 
+We charge EUR 3.000 per training day. 
+
+b) For closed trainings up to 30 attendees in Luxembourg: 
+We charge EUR 3.500 per training day including food and beverages
+
+
+## Support contract
+
+We offer a yearly support contract for the fast-tracked resolution of 5 issues.
+This contract includes the prioritised resolution of 5 issues, configuration requests or fast-tracked bug fixes for the MISP core software or modules maintained by us. The support is given through our ticket system within business hours.
+Fast-tracking a new/custom feature is usually project based and not included in this support contract, but depending on the complexity it could be handled within this contract. This kind of support costs EUR 6.000 per year.
+
+## Sustainability support
+
+In order to support the sustainability of MISP as an Open Source project, we offer the display of your logo prominently on the MISP Project website's supporter section. This kind of support costs EUR 9.000 per year.
+
+## Support contract with sustainability support
+
+In order to support the sustainability of MISP as an Open Source project, we offer the display of your logo prominently on the MISP Project website's supporter section. In addition, this contract includes the prioritised resolution of 6 issues, configuration requests or fast-tracked bug fixes for the MISP core software or modules maintained by us. The support is given through our ticket system within business hours.
+Fast-tracking a new/custom feature is usually project based and not included in this support contract, but depending on the complexity it could be handled within this contract. This kind of combined support costs EUR 15.000 per year.
+
+## Optional additional support offerings
+
+The following options are available for any holder of an active support contract:
+
+- Training, per day, on-site or in Luxembourg (EUR 2.000 per training day instead of EUR 3.000)
+- Installation support on-site or remotely (EUR 1.500 per day)
+- Integration support on-site or remotely (EUR 1.500 per day)
+- Development of new and/or custom features (project based, cost depending on scope)
+
+
+# Additional costs for all missions outside Luxembourg:
+
+We charge in addition to the displayed prices for the travel and accommodation costs according to the number of trainers or support engineers required.
+
+We might uplift reasonably depending on the training location and the time for travel.
```

Comment:
Based on the training location and the time of travel an additional uplift of the price may be included.